ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -877,25 +869,19 @@ tests:
   steps:
     cluster_profile: aws-qe
     env:
-      BASE_DOMAIN: qe.devcluster.openshift.com
-      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @proxy and @network-ovnkubernetes and
-        not @fips'
-      TAG_VERSION: '@4.14'
+      FEATURE_SET: TechPreviewNoUpgrade
     test:
     - chain: openshift-e2e-test-qe
-    workflow: cucushift-installer-rehearse-aws-ipi-proxy-techpreview-po
+    workflow: cucushift-installer-rehearse-aws-ipi-proxy-po
 - as: aws-ipi-proxy-techpreview-p3-f28-destructive
   cron: 15 19 16 * *
   steps:
     cluster_profile: aws-qe
     env:
-      BASE_DOMAIN: qe.devcluster.openshift.com
```

Comment:
@JoelSpeed same above comment. just need to add `FEATURE_SET: TechPreviewNoUpgrade`, no need to delete existing env.